### PR TITLE
Made Commands Case Insensitive and made it send message on getting pinged

### PR DESCRIPTION
**Pull request by TheEmperor342**

### CODE REVIEW

**Comment by itsmebirdie** [comment]
Acknowledged PR, review in progress

**Review by itsmebirdie — COMMENTED** [review]
Always have a professional commit description.

File: Cogs/TicTacToe.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ async def ttt(self, ctx: commands.Context, p2: commands.MemberConverter) -> None
     ## ==> TO ACCEPT A GAME
     ##############################################################################################################################################################
     
-    @commands.command(aliases=["Accept"])
+    @commands.command()
```
Comment (itsmebirdie) [inline Cogs/TicTacToe.py:113]: Doesn't this command require an alias?

**Review by itsmebirdie — COMMENTED** [review]
Review paused, awaiting comments on clarification. Testing in progress

File: Cogs/TicTacToe.py (hunk 1 above)
Comment (TheEmperor342) [reply Cogs/TicTacToe.py:113]: Nope it doesnt

**Comment by TheEmperor342** [comment]
> Always have a professional commit description.
I didnt knew that commit was gonna be mentioned here and I couldn't pull code too so yea

**Comment by itsmebirdie** [comment]
Approved

**Pull request merged**